macros/unpack/get_nodes.sql

```diff
@@ -25,7 +25,8 @@
                 "cast(" ~ dbt_project_evaluator.is_not_empty_string(node.description) | trim ~ " as boolean)",
                 "''" if not node.column_name else wrap_string_with_quotes(dbt.escape_single_quotes(node.column_name)),
                 wrap_string_with_quotes(node.meta | tojson),
-                wrap_string_with_quotes(node.depends_on.macros | tojson)
+                wrap_string_with_quotes(node.depends_on.macros | tojson),
+                wrap_string_with_quotes(node.test_metadata.name if node.test_metadata else ''),
```

Comment:
What if we just went ahead and calculated `is_generic_test` here instead of pulling in the `node.test_metadata.name` value?

Reply:
good question! mostly comes down to whether we think the test name (or other metadata) will be valuable in other contexts